app/code/core/Mage/Index/etc/config.xml

```diff
@@ -26,6 +26,18 @@
             <version>1.6.0.0</version>
         </Mage_Index>
     </modules>
+    <crontab>
+        <jobs>
+            <index_clean_events>
+                <schedule>
+                    <cron_expr>30 * * * *</cron_expr>
```

Comment:
```suggestion
                    <cron_expr>30 */4 * * *</cron_expr>
```
I'd make something like this if that's ok with you, to run every 4 hours, cause every 30 minutes it seems a bit too much no?